code/modules/paperwork/paper.dm

```diff
@@ -59,7 +59,7 @@
 	updateinfolinks()
 
 /obj/item/paper/oui_getcontent(mob/target)
-	if(!target.is_literate())
+	if(!target.is_literate() || isAI(target))
```

Comment:
This will also screw up the feature of holding up a paper on the camera for the AI to read it, keep in mind.

Reply:
This is kind of a weird way to do it to be perfectly honest though. I couldn't find a really more efficient way to do it with how oui works. Ideally I wanted a "force stars" variable in the proc but because of the way getcontent is called that's not possible without a lot of other changes.